CHANGELOG.md

```diff
@@ -1,41 +1,64 @@
-CHANGELOG
```

Comment:
One more non-coding remark: I found the previous version not readable, but I still find the new version complicated. I would use lists of short sentences (and no section).